src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figuresArray = new Figure[6];
+        for (int i = 0; i < figuresArray.length / 2; i++) {
```

Comment:
make `if` in `loop`

Reply:
for (int i = 0; i < figuresArray.length; i++) {
            if (i < figuresArray.length / 2) {
                figuresArray[i] = figureSupplier.getRandomFigure();
            }
        }
I don't know why this need